pkg/config/config.go

```diff
@@ -18,6 +18,7 @@ type Config struct {
 	// This should be an instance of deploy.FeatureMap which is a specific json format for this
 	// Use deploy.UnmarshalFeatureMap to unmarshal this data into a FeatureMap
 	IdentitySettings string `env:"IDENTITY_SETTINGS,default={}"`
+	KMSKey           string `env:"AWS_KMS_KEY,required"`
```

Comment:
Keep the naming consistent
You also need to add this KMS_KEY_ARN env var to the lambda definitions which require it.
Currently it will only work in dev, make sure you test this on your deployed stack too
```suggestion
	KMSKeyARN           string `env:"KMS_KEY_ARN,required"`
```